x-pack/platform/plugins/shared/streams_app/ui_tests/README.md

```diff
@@ -0,0 +1,25 @@
+## How to run tests
+
+First start the servers:
+
+```bash
+// ESS
+node scripts/scout.js start-server --stateful
+
+// Serverless
+node scripts/scout.js start-server --serverless=[es|oblt|security]
+```
+
+Then you can run the tests in another terminal:
+
+Some tests are designed to run sequentially:
+
+```bash
+// ESS
+npx playwright test --config x-pack/platform/plugins/shared/streams_app/ui_tests/playwright.config.ts --project=local --grep @ess
+
+// Serverless
+npx playwright test --config x-pack/platform/plugins/shared/streams_app/ui_tests/playwright.config.ts --project=local --grep @svlOblt
+```
+
+Test results are available in `x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/output`
```

Comment:
We have a reference to `observability_onboarding` here that we may want to remove.